### DISCUSSION TITLE
Share audio between two computers

### DISCUSSION THREAD

**Discussion by pedrobzz** [debauchee/barrier#1457] [Ideas]
This is probably not an innovative idea, it is just something that would help me a LOT since I use the barrier to have a "second monitor". I used to use spacedesk for this task but the image looks horrible, and since I do some front end development it is very bad to have a screen with distorted colors and low resolution. Barrier, on the other hand, fulfills this role well, and the only missing feature, in my opinion, is the possibility to share the audio beyond mouse and keyboard

**Comment by Zamana** [comment]
Yes. The possibility of having the remote audio in the environment from where you are operating the whole system is mandatory in my case.
